### DISCUSSION TITLE
"bit  remote add" command fails with "remote scope protocol is not supported" on Mac

### DISCUSSION THREAD

**Discussion by alexf2** [teambit/bit#7526] [Help]
Just have installed bit on Mac, trying to add a remote scope, but Bit fails with: 

  error: remote scope protocol is not supported, please use: `ssh://`, `file://` or `bit://`.

Has someone encountered the same issue?

**Comment by itaymendel** [comment]
hi, we miss additional context on how you set your scope.
have you consulted the docs? https://bit.dev/reference/scope/configure-scopes
